WooCommerce/WooCommerceTests/System/SessionManagerTests.swift

```diff
@@ -109,6 +109,27 @@ class SessionManagerTests: XCTestCase {
         }
     }
 
+    /// Verifies that `completedAllStoreOnboardingTasks` is set to `nil` upon reset
+    ///
+    func test_completedAllStoreOnboardingTasks_is_set_to_nil_upon_reset() throws {
+        // Given
+        let uuid = UUID().uuidString
+        let defaults = UserDefaults(suiteName: uuid)!
```

Comment:
nit: this can use try unwrapping to avoid a crash the test suite if it fails to initialize
```suggestion
        let defaults = try XCTUnwrap(UserDefaults(suiteName: uuid))
```

Similar for other places in the PR

Reply:
Done in bc00a81fd2ca75564339f33cbf18d1206cee2057